src/components/ButtonPanel.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import Button from './Button'
+
+function ButtonPanel() {
+  return (
+    <div className="button-panel">
```

Comment:
- [ ] Here at the ButtonPanel component we need to use the Button component instead of the button element:
```JSX
        <Button name="7" />
        <Button name="8" />
        <Button name="9" />
        <Button name="X" />
```

- [ ] I think you used `names` here (plural) instead of `name` as it is declared in the Button component.